core/src/Tabster.ts

```diff
@@ -34,6 +34,7 @@ class Tabster implements Types.TabsterCore, Types.TabsterInternal {
     private _win: WindowWithTabsterInstance | undefined;
     private _forgetMemorizedTimer: number | undefined;
     private _forgetMemorizedElements: HTMLElement[] = [];
+    public _version: string | undefined = __VERSION__;
```

Comment:
Any specific reason to have | undefined? I suppose it should always be defined.
Also I suppose all our WebPack watchers (storybook and test container) should be adjusted to process this variable.

Reply:
not really, it's a good idea to update all our webpacks with this